model/src/main/scala/com/lightbend/scala/modelServer/model/Model.scala

```diff
@@ -5,7 +5,7 @@ package com.lightbend.scala.modelServer.model
  * Basic trait for model
  */
 trait Model {
-  def score(input: AnyVal): AnyVal
+  def score(input: Any): Any
```

Comment:
I'd prefer real types here, as type parameters if it should support different things.
Anyway, the AnyVal was probably not right since for example input was WineRecord.

Reply:
AnyVal (or Any) is here to make this interface completely generic. I did not want to introduce specific interface for WineRecord and change it every time data model changes.

Reply:
Then it's Any, not AnyVal